tests/airgapped/setup/setup.sh

```diff
@@ -2,6 +2,8 @@
 set -xe
 
 cat tests/airgapped/lxd.profile | lxd init --preseed
+sudo apt-get update
+sudo apt-get -y install python3-pip
```

Comment:
@NohaIhab I see why you added this in the first place. In my case I must had re-run the script at some point and had `pip3` installed.

But, we install `python3-pip` in another place, in https://github.com/canonical/bundle-kubeflow/blob/e4eadef12bc1295a505b89b257413053a783772c/tests/airgapped/setup/prerequisites.sh#L5

Also, I realised that the `scripts/airgapped/requirements.txt` are used already in
https://github.com/canonical/bundle-kubeflow/blob/e4eadef12bc1295a505b89b257413053a783772c/scripts/airgapped/prerequisites.sh#L7

So I suggest we do the following changes to this file:
```bash
#!/usr/bin/env bash
set -xe

cat tests/airgapped/lxd.profile | lxd init --preseed

./tests/airgapped/setup/prerequisites.sh
./scripts/airgapped/prerequisites.sh
./tests/airgapped/setup/lxd-docker-networking.sh
```

Which essentially first runs the `scripts/airgapped/setup/prerequisites.sh` file, which downloads pip3, and then the rest of the scripts will install the other dependencies